app-modules/service-management/src/Filament/Resources/ServiceRequestUpdateResource/Pages/ViewServiceRequestUpdate.php

```diff
@@ -77,7 +78,7 @@ public function infolist(Infolist $infolist): Infolist
     protected function getHeaderActions(): array
     {
         return [
-            Actions\EditAction::make(),
+            Actions\EditAction::make()->visible(fn ($record) => $record?->serviceRequest?->status?->classification == SystemServiceRequestClassification::Closed ? false : true),
```

Comment:
Is this `visible` check needed now that you have applied the same logic to the Policy `update` method?

I could be wrong, but I think the default `EditAction` may check that Policy method.